blocks/article-body-block/chains/article-body/default.jsx

```diff
@@ -63,7 +64,13 @@ function parseArticleItem(item, index, arcSite, phrases, id, customFields) {
 		}
 
 		case "divider": {
-			return <hr className={`${BLOCK_CLASS_NAME}__divider`} key={`${type}_${index}_${key}`} />;
+			return (
+				<Divider
+					assistiveHidden={false}
```

Comment:
Should this be false?

Reply:
It's mentioned in the Acceptance criteria

Reply:
We are setting this `false` here as we are making the assumption authors are using the divider in composer in the correct way.